tests/arrays/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -4,8 +4,10 @@ exports[`empty.js 1`] = `
 const a = someVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeLong.Expression || [];
 const a = someVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeLong.Expression || {};
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-const a = someVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeLong.Expression || [];
-const a = someVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeLong.Expression || {};
+const a =
+  someVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeLong.Expression || [];
+const a =
+  someVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeLong.Expression || {};
```

Comment:
This was actually intentional, I wanted to make sure that the following pattern ( https://github.com/prettier/prettier/pull/1590#discussion_r116126351 ) is printed in one line. The pattern in this comment is almost never used so it doesn't really matter if it is > 80 columns. Also putting it on the next line isn't really pretty either.